h/accounts/schemas.py

```diff
@@ -323,6 +340,50 @@ def validator(self, node, value):
             raise exc
 
 
+def validate_url(value):
+    return wrap_validator(util.validate_url)(value)
```

Comment:
It would probably be a little neater to use the `wrap_validator` function you've written as a decorator. You can actually do this right now:

```python
@wrap_validator
def validate_url(value):
    <body of util.validate_url>
```

But that wouldn't be ideal, as debugging the resulting `validate_url` would be tricky due to the way the decorator would change the function. To fix that you'd add one line to your `wrap_validator` decorator definition:

```python
import functools

def wrap_validator(validator):
    @functools.wraps(validator)
    def validate(value):
        ...
    return validate
```